scripts/info.mumble.Mumble.appdata.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2021      Robert Adam -->
+<!-- Copyright 2019-2022 Jonah Brüchert -->
```

Comment:
If that's okay with you, I'd prefer a generic license header (as we use on our other source files) over individual copyright notices.